src/components/instructor/AssignmentGrade.js

```diff
@@ -1,21 +1,89 @@
-import React, { useState } from 'react';
-
-// instructor enters students' grades for an assignment
-// fetch the grades using the URL /assignment/{id}/grades
-// REST api returns a list of GradeDTO objects
-// display the list as a table with columns 'gradeId', 'student name', 'student email', 'score' 
-// score column is an input field 
-//  <input type="text" name="score" value={g.score} onChange={onChange} />
- 
-
-const AssignmentGrade = (props) => {
-
- 
-    return(
-        <>
-            <h3>Not implemented</h3>
-        </>          
+import React, { useState, useEffect } from 'react';
+import { SERVER_URL } from '../../Constants';
+
+const AssignmentGrade = ({ assignmentId }) => {
+    const [grades, setGrades] = useState([]);
+    const [message, setMessage] = useState('');
+    const instructorEmail = 'dwisneski@csumb.edu';
+
+    useEffect(() => {
+        fetchGrades();
+    }, [assignmentId]);
+
+    const fetchGrades = async () => {
+        try {
+            const response = await fetch(`${SERVER_URL}/assignment/${assignmentId}/grades?email=${instructorEmail}`);
+            if (response.ok) {
+                const data = await response.json();
+                setGrades(data);
+            } else {
+                const errorData = await response.json();
+                setMessage(`Error: ${errorData.message}`);
+            }
+        } catch (error) {
+            setMessage(`Network error: ${error.message}`);
+        }
+    };
```

Comment:
There is an issue in here that's causing a network error on my end. We managed to resolve it on the other branch, so we are going to decline to merge.